cartoframes/data/observatory/variable.py

```diff
@@ -48,3 +48,18 @@ def starred(self):
     @property
     def summary(self):
         return self.data['summary_jsonb']
+
+    @property
+    def project_name(self):
+        project, _, _, _ = self.id.split('.')
```

Comment:
maybe we can do the split once in the init, setting project, schema and dataset as private props of the class and retuning them in these methods.